enterprise-suite/values.yaml

```diff
@@ -23,3 +23,6 @@ imagePullPolicy: IfNotPresent
 prometheusDomain: prometheus.io
 # minikube debug resources
 minikube: true
+# console server type
+esServiceType: ClusterIP
```

Comment:
I've said this many times, but bear with me as I repeat:

    Please treat NodePort as a security hole.

We should _not_ install it.  If customer needs to install it for some vendor specific reason in a circumscribed manner and on their (not our) recommendation, hey they can do that.  But we should treat it as an electric fence not to be touched.

https://oteemo.com/2017/12/12/think-nodeport-kubernetes/